functions/index.js

```diff
@@ -11,6 +11,8 @@ let isReady = false;
 
 let nuxt = loadNuxt('start');
 
+let dynamicRoutes = ['/_ghapi/status', '/_workday/opportunities']
```

Comment:
I'd consider renaming this to whitelistedRoutes, that way we know that these routes are allowed by the cloud function. It's the normal language used for this sort of thing in networking. Obviously this doesn't matter either ;-)

Reply:
I don’t love the whitel/black list terminology, but I could use allowedRoutes or something more clear